components/buy-button.tpl

```diff
@@ -45,9 +47,7 @@
           <span class="edy-buy-button-title">{{ 'add_to_cart' | lc | escape_once }}</span>
         {% endif -%}
 
-        {%- if with_price and product.price_min_with_tax == product.price_max_with_tax -%}
-          <span class="edy-buy-button-price">{{ product.price_with_tax | money_with_currency: product.currency }}</span>
-        {%- endif -%}
+      <div class="edy-buy-button-price-container"></div>
```

Comment:
Due to buy button structure changes, this div has to be outside of `edy-buy-button` container. Currently prices appear inside buy button on auto-gen product pages.

<img width="479" alt="Screenshot 2023-11-21 at 12 17 18" src="https://github.com/Voog/design-firenze/assets/44435196/f8e8ae73-d2d9-4ae0-8728-c26aaa73e515">